File: docker/languages/Dockerfile.cpp
# Utilise une image de base avec GCC
FROM gcc:latest

# Définis le répertoire de travail
WORKDIR /app

# Copie un script de test simple pour vérifier le fonctionnement
COPY /docker/languages/test_cpp.cpp .

# Commande par défaut (peut être écrasée)
CMD g++ -o test_cpp test_cpp.cpp && ./test_cpp

File: docker/languages/Dockerfile.java
# Utilise une image OpenJDK
FROM openjdk:11-jdk-slim

WORKDIR /app

# Comme nous sommes déjà dans docker/languages, juste copier TestJava.java
COPY TestJava.java .

CMD ["sh", "-c", "javac TestJava.java && java -cp . TestJava"]

